src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Stores/OpenIdAuthorizationStore.cs

```diff
@@ -59,60 +61,53 @@ public virtual async ValueTask DeleteAsync(TAuthorization authorization, Cancell
     }
 
     /// <inheritdoc/>
-    public virtual IAsyncEnumerable<TAuthorization> FindAsync(
-        string subject, string client, CancellationToken cancellationToken)
+    public virtual async IAsyncEnumerable<TAuthorization> FindAsync(
+        string subject, string client, string status, string type,
```

Comment:
It's not obvious since we're not using nullable annotations here, but these parameters are now nullable in 6.0 preview3 (which gives a lot more flexibility to filter the authorizations/tokens).